2025-08-09T05:40:09.415Z [ERROR]: Error: Testing the global error handler
    at C:\Users\91869\OneDrive\Desktop\Namaste Node\project\devtinder_backend\devtinder\src\app.js:61:8
    at Layer.handleRequest (C:\Users\91869\OneDrive\Desktop\Namaste Node\project\devtinder_backend\devtinder\node_modules\router\lib\layer.js:152:17)
    at next (C:\Users\91869\OneDrive\Desktop\Namaste Node\project\devtinder_backend\devtinder\node_modules\router\lib\route.js:157:13)
    at Route.dispatch (C:\Users\91869\OneDrive\Desktop\Namaste Node\project\devtinder_backend\devtinder\node_modules\router\lib\route.js:117:3)
    at handle (C:\Users\91869\OneDrive\Desktop\Namaste Node\project\devtinder_backend\devtinder\node_modules\router\index.js:435:11)
    at Layer.handleRequest (C:\Users\91869\OneDrive\Desktop\Namaste Node\project\devtinder_backend\devtinder\node_modules\router\lib\layer.js:152:17)
    at C:\Users\91869\OneDrive\Desktop\Namaste Node\project\devtinder_backend\devtinder\node_modules\router\index.js:295:15
    at processParams (C:\Users\91869\OneDrive\Desktop\Namaste Node\project\devtinder_backend\devtinder\node_modules\router\index.js:582:12)
    at next (C:\Users\91869\OneDrive\Desktop\Namaste Node\project\devtinder_backend\devtinder\node_modules\router\index.js:291:5)
    at Function.handle (C:\Users\91869\OneDrive\Desktop\Namaste Node\project\devtinder_backend\devtinder\node_modules\router\index.js:186:3)
